src/components/Button/IconButton/IconButton.tsx

```diff
@@ -50,9 +50,9 @@ type IconButtonProps = UnstyledButtonPropsFor<"button"> &
      */
     destructive?: boolean;
     /**
-     * The label for the button
+     * Display a tooltip when the button is hovered
```

Comment:
I think `is hovered` is misleading here, because the tooltip will also show for keyboard accessibility users which have no pointer/hover means

Reply:
Rhaaaa, what do you propose instead?

Reply:
Don't need to specify the semantics of when it'll show;

`Optional tooltip for the button`